src/work-allocation/guards/tribunal-caseworker-guard.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, Router } from '@angular/router';
+import { UserInfo } from '../../app/models/user-details.model';
+import { SessionStorageService } from '../../app/services';
+
+@Injectable()
+export class TribunalCaseworkerGuard implements CanActivate {
+    public static defaultUrl: string = '/cases';
+    public static caseWorkerRole: string = 'caseworker-ia-admofficer';
+    public static seniorCaseWorkerRole: string = 'caseworker-ia-caseofficer';
```

Comment:
Seeing as this is also in the `senior-tribunal-caseworker-guard`, it might be sensible to put these role constants in a separate file so they can't get out of sync.

Also, I _think_ lint will moan at you about the `: string` part as being "trivial". Not sure when it comes to `static`s but it definitely moans at me for instance variables.

Reply:
The reason I wanted to Static is that they are technically not instance dependant.